src/gui/src/dbDescriptors.cpp

```diff
@@ -530,10 +530,36 @@ bool DbNetDescriptor::getBBox(std::any object, odb::Rect& bbox) const
 {
   auto net = std::any_cast<odb::dbNet*>(object);
   auto wire = net->getWire();
-  if (wire && wire->getBBox(bbox)) {
-    return true;
+  bool has_box = false;
+  bbox.mergeInit();
+  if (wire) {
+    odb::Rect wire_box;
+    if (wire->getBBox(wire_box)) {
+      bbox.merge(wire_box);
+      has_box = true;
+    }
   }
-  return false;
+
+  for (auto inst_term : net->getITerms()) {
+    if (!inst_term->getInst()->getPlacementStatus().isPlaced()) {
+      continue;
+    }
+
+    odb::dbBox* term_bbox = inst_term->getInst()->getBBox();
```

Comment:
If the net connects to a macro this will oversize the bbox.  Use dbITerm::getAvgXY instead

Reply:
That should have been the BBox of the iterm.